cfe_internal/update/update_processes.cf

```diff
@@ -154,12 +162,17 @@ bundle agent maintain_cfe_hub_process
       handle => "cfe_internal_maintain_cfe_hub_process_processes_postgres",
       ifvarclass => "nova|enterprise";
 
-    am_policy_hub.files_ok.!windows::
+    am_policy_hub.!enable_cfengine_enterprise_hub_ha.files_ok.!windows|ha_run_hub_process::
       "cf-hub"      restart_class => "start_hub",
       comment => "Monitor cf-hub process",
       handle => "cfe_internal_maintain_cfe_hub_process_processes_cf_hub",
       ifvarclass => "(nova|enterprise).no_vacuumdb";
 
+    am_policy_hub.ha_kill_hub_process::
+      "cf-hub"      signals => { "term" },
+      comment => "Terminate cf-hub on backup HA node outside cluster",
```

Comment:
In order to not try contacting agents and try to collect reports. If we will have more than one hub collecting reports we will surely end up with mess.